### ISSUE TITLE
Verwaltung der eingegebenen Daten

### ISSUE THREAD

**Issue by a71na** [IU-Internationale-Hochschule-Augsburg/fallstudie-thema-2-online-umfragesystem#3]
Wenn Daten über das Online-Umfragesystem eingegeben werden, soll das System die Daten entsprechend in einer Datenbank speichern und verwalten.

Dabei ergibt sich die Frage, welche Datenbank wir verwenden möchten? (MySQL, PostgreSQL, ...)

**Comment by d00mfry** [comment]
SQLite würde ggf. auch ausreichen, wir fliegen ja mit dem Ding nicht zum Mond. Aber ich kann schlecht abschätzen inwiefern es problematisch wäre, wenn das Umfragetool doch komplexer wird als gedacht. 

Pro: SQLite ist leichtgewichtig
Contra: Skaliert schlecht, bei uns aber mMn unbedeutend

**Comment by a71na** [comment]
Ja, neben SQLite gibt es natürlich auch noch weitere embedded Datenbanken wie bspw. H2. Aber das müssen wir übermorgen nochmal genauer besprechen :).
